portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,21 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const facts =
+      [
+      'I was born on April 24th, 2000.',
+      'I learned Spanish before I learned English.',
+      'I have not cut the length of my hair since I graduated high school two years ago.', 
+      'I was named after Sean Connery, but I do not believe I have ever seen any of his movies.'
+      ];
```

Comment:
Try something like this instead for formatting. It's more common in Google code. Also, add a trailing comma to the last element in the list.

```suggestion
  const facts = [
      'I was born on April 24th, 2000.',
      'I learned Spanish before I learned English.',
      'I have not cut the length of my hair since I graduated high school two years ago.', 
      'I was named after Sean Connery, but I do not believe I have ever seen any of his movies.',
  ];
```